crates/bevy_render/macros/src/lib.rs

```diff
@@ -17,6 +18,37 @@ pub fn derive_extract_resource(input: TokenStream) -> TokenStream {
     extract_resource::derive_extract_resource(input)
 }
 
+/// Implements `ExtractComponent` trait for a component.
```

Comment:
Since `bevy_render/macros` is a "downstream" crate, is there any way for me to actually make proper links in the docs here?

Reply:
No good way: you can link to the webpage 🙃 

Don't worry about it IMO.